### ISSUE TITLE
https://reddit-top-rss.herokuapp.com - Application Error

### ISSUE THREAD

**Issue by markus1189** [johnwarne/reddit-top-rss#16]
It seems like the application has an error on heroku.

Do you plan to fix the error, or is it just no longer being maintained?

As much as I enjoy the free instance, I can understand if the latter is the case, just wanted to check before thinking of a solution.

**Comment by daddy32** [comment]
Heroku ended the free hosting plans after "28th of november" (actually they've added a few more days to the grace period), so I guess this is what's happened to the instance. To continue with running it, the minimum price asked by heroku is $5...

**Comment by glottisfaun0000** [comment]
in the meantime while we all figure out new solutions, i learned today that adding ?t=week after .rss to a subreddit rss narrows the results so you can approximate the low-volume that this app produces. example: old.reddit.com/r/Gradschool/top/.rss?t=month . also gives you old.reddit.com results which reddit-top-rss can't

**Comment by markus1189** [comment]
I am currently running the docker container locally on my home-server which is fine for me, because I ssh into it and use [newsboat](https://newsboat.org/) via terminal anyway.

It's not a good solution if you are reading RSS from e.g. feedly or some device that is not always on and on the same network as the docker container...

**Comment by daddy32** [comment]
Also, there is this alternative: https://old.reddit.com/r/rss/comments/fvg3ed/i_built_a_better_rss_feed_for_reddit/
Not as feature-packed (~~and cannot point to old.reddit~~), but works, points to old.reddit by defaul and has a free instance (at least for now).

**Comment by glottisfaun0000** [comment]
> Also, there is this alternative: https://old.reddit.com/r/rss/comments/fvg3ed/i_built_a_better_rss_feed_for_reddit/
> Not as feature-packed (and cannot point to old.reddit), but works and has a free instance (at least for now).

This one is good, and it actually points to old.reddit by default: https://github.com/trashhalo/reddit-rss#reddit_url

**Comment by daddy32** [comment]
You're right! Great.

**Comment by aderchox** [comment]
@glottisfaun0000 Thanks but it seems that [it's buggy](https://www.reddit.com/r/bugs/comments/108cq9z/posts_in_rss_feed_and_in_actual_feed_dont_match/) or at least I'm missing something.

**Comment by daddy32** [comment]
> @glottisfaun0000 Thanks but it seems that [it's buggy](https://www.reddit.com/r/bugs/comments/108cq9z/posts_in_rss_feed_and_in_actual_feed_dont_match/) or at least I'm missing something.

No, that's a bug for the official feed if I'm reading correctly. Try this one:
https://github.com/trashhalo/reddit-rss#using-my-free-hosted-version

**Comment by teeann** [comment]
[Render](https://render.com/) is a good alternative to Heroku. On Render, I was able to deploy this web app for free in less than 5 minutes, so please give it a try!
